nupic/regions/CLAClassifierRegion.py

```diff
@@ -224,20 +278,58 @@ def compute(self, inputs, outputs):
     Process one input sample.
     This method is called by the runtime engine.
 
-    We don't use this method in this region because the inputs and outputs don't
-    fit the standard "vector of reals" used by the engine. Instead, call
-    the customCompute() method directly
     """
 
-    pass
+    # An input can potentially belong to multiple categories. 
+    # If a category value is < 0, it means that the input does not belong to that category.
+    categories = []
+    for category in inputs['categoryIn']:
+      # if a category value <0, then it means 
+      # the input record does not belong to that category.
+      if category >= 0:
+        categories.append(category)
```

Comment:
Could be rewritten as a list comprehension:

`categories = [category for category in inputs["categoryIn"] if category >= 0]`